proto/data.go

```diff
@@ -316,6 +317,69 @@ func (v *Value) SetProto(msg gogoproto.Message) error {
 	return nil
 }
 
+// SetTime encodes the specified time value into the bytes field of the
+// receiver and sets the tag.
+func (v *Value) SetTime(t time.Time) error {
+	var err error
+	if v.Bytes, err = t.MarshalBinary(); err != nil {
```

Comment:
that would be more consistent considering that we at times read values from the index, and at other times from the payload.